modules/applications/applications-nav.adoc

```diff
@@ -31,6 +31,7 @@
     ***** xref:ui-builder/common-apis-to-use.adoc[Common APIs to use]
     ***** xref:ui-builder/how-to-retrieve-user-avatar.adoc[Retrieve user avatar image]
     ***** xref:ui-builder/reuse-code-and-components.adoc[Reuse code and components]
+    ***** xref:ui-builder/create-fully-responsive-apps.adoc[Ensure the application is fully responsive]
```

Comment:
we can challenge `fully`

Reply:
I would be agree with you, but when changing that, I see that one can have the feeling that the app built with Appsmith is not responsive by default... Which is not the case. So finally I prefer to keep it like it is.